### DISCUSSION TITLE
Daylight savings time / Time change

### DISCUSSION THREAD

**Discussion by Jkh62** [jgyates/genmon#1084] [General]
Does the time change need to be done at the generator (Generac controller ) or can it be changed thru the genmon interface? Just realized that we had the DST change and my Gen must be an hour behind since no exercise email. I remember seeing a "set time, date" but am hesitant to click things I am not sure what it does. 

I wish they would pick a time and leave it alone.....

**Comment by jgyates** [comment]
There are two settings on the settings page that should answer your question. If you hover your mouse over each setting you will get a description of what each setting does. 

The raspberry pi OS by default will sync the Pi's time to a time server on the Internet. You can set you generator time to the same time as the pi by clicking the "Set Generator Time" button on the Maintenance page. If you enable the "Sync Time" option on the settings page, genmon will update to generator time to the pi time every hour. If you enable "Sync Daylight Savings Time" option on the settings page, the generator time will be updated when the pi automatically adjusts it's time for DST.

Most controllers do not support settings the time seconds so if you set the generator  time to match the pi it will be within on minute of the pi's time.

Of course this only applies if your controller supports setting the time over modbus. Generac models do support this as well as some of the custom controllers.

**Comment by Jkh62** [comment]
Still learning / thx for the info.
